views/artist-search-results.hbs

```diff
@@ -0,0 +1,13 @@
+{{#each artistsList}}
+<img src="{{images.[0].url}}" alt="">
+<p>
+    {{!-- {{#each this.images}}
+{{url}}
+    {{/each this.images}} --}}
+</p>
+<p>
+    {{#each this.name}}
+    {{/each}}
+</p>
+
```

Comment:
You don't need to iterate through this.name, each artist has one name.

you only need {{name}}